notebooks/official/ray_on_vertex_ai/get_started_with_pytorch_rov.ipynb

```diff
@@ -0,0 +1,1646 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    ! pip3 install {USER} google-auth==2.27.0 etils==1.6.0 -q --no-warn-conflicts
</pre><p>If the version for etils is no significant, can we not pin the version or pin to one of these (1.5.2 or 1.5.1)</p><p> </p><pre class="ql-syntax" spellcheck="false">There was an error in the integration test: ERROR: No matching distribution found for etils==1.6.0
</pre><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2711/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
done